packages/cspell-lib/src/Settings/CSpellSettingsServer.ts

```diff
@@ -242,13 +243,21 @@ function applyPatterns(regExpList: (string | RegExp)[] = [], patterns: RegExpPat
     return regExpList.map(p => patternMap.get(p.toString().toLowerCase()) || p);
 }
 
-const testNodeModules = /^node_modules\//;
+const nodeModulesString = 'node_modules/';
 
 function resolveFilename(filename: string, relativeTo: string) {
-    if (testNodeModules.test(filename)) {
-        filename = require.resolve(filename.replace(testNodeModules, ''));
+    if (filename.includes(nodeModulesString)) {
```

Comment:
I need to think about this. It is a change of behavior and prevents absolute paths to node_modules.

Reply:
That's a good point. I'll make adjustments to my code.